doc/content/examples/example9.md

```diff
@@ -0,0 +1,95 @@
+
+### Individual soil response
+
+The input file for simulating the individual soil response is presented below. The uniform soil domain has a shear wave velocity of $165$ $m/s$. Although the soil is uniform throughout, the `UniformLayerAuxKernel` is used to specify the top surface of the soil. This is necessary when using the `ComputeIsotropicElasticityTensorSoil` material. Alternatively, if the `ComputeIsotropicElasticityTensor` is used for a uniform soil, the `UniformLayerAuxKernel` is not necessary. In addition, periodic boundary conditions are applied to the two opposite faces of the soil domain which are perpendicular to the Z axis to ensure that they move in unison. This achieved using the `Periodic` option in `BCs`. The Ormsby Wavelet is applied at the bottom of the soil block and the response is recorded at the top. The 1% damped acceleration response spectrum at the top of the soil is presented below. The periods at which the response spectrum peaks are the fundamental periods of the soil domain. The periods inferred from this figure are very similar to those predicted by the theoretical solution $4H/((2n-1)~Vs)$, where $H$ is the depth of soil and $n$ is the index of the period of interest.
+
+!listing examples/ex09/Soil_Response.i
+
+!media media/examples/ex09_Soil.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Soil
+       caption=Response spectra of the individual soil response under the Ormsby Wavelet.
+
+### Soil Structure Interaction response
+
+The input file for simulating the SSI response is presented below. The amplification ratio of the structural response spectra from the free-field soil is computed. Free-field soil is the soil region located far away from the structure, which in this case is one of the top end most nodes of the domain. This amplification ratio as a function of the period is presented below. Also presented below is the amplification ratio obtained by [!citet](BolisettiThesis) using LS-Dyna. It is seen that the variation of the amplification ratios with period is similar between MASTODON and LS-Dyna.
+
+!listing examples/ex09/SSI_Response.i
+
+!row!
+
+!col! small=12 medium=6 large=6
+!media media/examples/ex09_SSI.png
+       style=width:100% id=fig:SSI_Amp
+       caption=Amplification ratio of the structural response spectra from the free-field soil in SSI analysis. This is computed using MASTODON.
+!col-end!
+
+!col! small=12 medium=6 large=6
+!media media/examples/ex09_SSI_Valid.png
+       style=width:100% id=fig:SSI_Amp_Valid
+       caption=Amplification ratio computed using LS-Dyna by [!citet](BolisettiThesis).
+!col-end!
+
+!row-end!
+
+Additionally, the 1% acceleration response spectrum of the free-field soil response from SSI analysis is presented below. Along with it, the response spectrum obtained from the individual soil response is also plotted. It is seen that these to spectra are very similar.
```

Comment:
Figure 8 shows the free-field soil acceleration response (1% damped) from the SSI analysis, compared to that obtained from the site response analysis.